HW1/P2/P2a.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
+from pyspark import SparkContext
 
-# Your code here
+sc = SparkContext("local", "Simple App")
+
+array_1 = [i for i in range(2000)]
+array_2 = [i for i in range(2000)]
+
+new_array = []
+for i in array_1:
```

Comment:
In Problem 2a, using Cartesian may be a good alternative to initialize this large coordinate array. Otherwise, "new_array" will not be parallelized at the beginning but rather stored in memory.